### DISCUSSION TITLE
Is it possible to change colors without ruining images?

### DISCUSSION THREAD

**Discussion by taltamir** [sumatrapdfreader/sumatrapdf#2360] [Q&A]
I can use advanced setting to change the color scheme of sumatraPDF to have a dark theme (black background with bright text).
However doing so also inverts all embedded images. making them look bad.

Is it possible to change the text colors only without altering embedded images in PDF?

**Comment by kjk** [comment]
It is not possible. Due to how PDF format works, inverting the rendered PDF page after it's rasterization as an image is the best we can do without spending unreasonable amount of engineering.

Note: I've edited out profanity from your question but I'll ban people who can't participate in a civil way.

**Comment by taltamir** [comment]
> It is not possible. Due to how PDF format works, inverting the rendered PDF page after it's rasterization as an image is the best we can do without spending unreasonable amount of engineering.

alright. thank you.
what about displaying it as a comic book? I noticed that there are setting for "fixed page" "ebook" and "comic book" in the advanced setting. How do I switch to comic book display? do I rename the files into a different extension? I looked everywhere and could not find it

**Reply by GitHubRulesOK** [reply]
Comic book is images in a zip so fairly easy to convert pdf to comic book (if not the best reading format) can be done easy in one or two lines in cmd or batch file.
***xpdf*** or poppler or mupdf all have simple pdfTOpng (or equivalent) ensure they are numbered something001.png something 002.png .... etc.
then using win 10 cmd **TAR -a -m**  blah blah (or other zip archive maker) the second line would need to add all the numbered images into a .cbz file
The searchable nature of PDF will be lost for those with text even though some attempts were made to add text to comic books the related problems were never addressed well enough, and djvu was the nearest to suceed but is only just hanging on as an also ran format even XPS could not oust PDF and only ebooks like ePub are a hybrid competiotor to PDF.

**Reply by taltamir** [reply]
I meant if there was an option to switch the view mode to "comic book view" on a PDF file.
But from the various answers displayed thus far I am going to take it as a no. I need to get the original images and convert them to a cb7 file.

thank you

**Comment by taltamir** [comment]
I just figured out how to see previous edits made so I can tell what the issue was now. well, i will be more careful with my wording then.
